extensions/cornerstone/src/utils/segmentUtils.ts

```diff
@@ -0,0 +1,45 @@
+import { servicesManager } from '@ohif/app/src/App';
+
+const {
+  segmentationService,
+} = servicesManager.services;
```

Comment:
We shouldn't import like this as far as i know, pass in the segmentationService from the component itself like onSegmentChange(options) and options has those + segmentationService